plugins/modules/zos_fetch.py

```diff
@@ -170,66 +167,66 @@
 
 RETURN = r"""
 file:
-    description: The source file path on remote machine
+    description: The source file path or data set on the remote machine.
     returned: success
     type: str
     sample: SOME.DATA.SET
 dest:
-    description: The destination file path on controlling machine
+    description: The destination file path on the controlling machine.
     returned: success
     type: str
     sample: /tmp/SOME.DATA.SET
 is_binary:
-    description: Indicates which transfer mode was used to fetch the file
+    description: Indicates the transfer mode that was used to fetch.
     returned: success
     type: bool
     sample: True
 checksum:
-    description: The SHA256 checksum of the fetched file
+    description: The SHA256 checksum of the fetched file.
```

Comment:
I am not really clear on when this is returned, i see it says file , i see if its a non-partitioned data set, i also see that the checksum description says:
```  validate_checksum:
    description:
      - Verify that the source and destination checksums match after the files
        are fetched.``` but it says files does this work data sets ? Which ones? 

I also see
``` - To prevent redundancy, additional checksum validation will not be done
      when fetching PDS(E) because data integrity checks are done through the
      transfer methods used. As a result, the module response will not include
      C(checksum) parameter.``` how about PDSE members, PDS, etc, i think we need some clarity around this.

Reply:
@ddimatos @RadhaV2020 - I should have been clearer about this. Just to clear things up:
1. `checksum` is returned when fetching USS files or sequential data sets.
2. `checksum` is not calculated for PDS/PDSE or PDS/PDSE members.